packages/react-i18n/src/utilities/decimals.json

```diff
@@ -0,0 +1,174 @@
+{
```

Comment:
Can we make it a Map to follow what we have in the [`constants` file](https://github.com/Shopify/quilt/blob/master/packages/react-i18n/src/constants.ts#L42)?
And could you move that file into a `constant` folders actually? 🙏